pkg/build/controller/controller.go

```diff
@@ -220,7 +223,7 @@ func (bc *BuildPodController) CancelBuild(build *buildapi.Build, pod *kapi.Pod)
 		return err
 	}
 
-	glog.V(2).Infof("Build %s was successfully cancelled.", build.Name)
+	glog.V(2).Infof("Build %s/%s was successfully cancelled.", build.Namespace, build.Name)
```

Comment:
+1 for doing this. But would rather make a separate PR for this, after this is merged. After that I can do through the origin code and can change it on suitable places.
Thoughts ?